cypress/private/cypress_module_test.bzl

```diff
@@ -0,0 +1,47 @@
+load("@aspect_rules_js//js:libs.bzl", "js_binary_lib", "js_lib_helpers")
+
+_attrs = js_binary_lib.attrs
+
+def _impl(ctx):
```

Comment:
The rule implementations of the two flavors look identical. Is the only difference in the macros?

Reply:
I can't believe I didn't see that myself. I just thought they were very similar, not identical :p

Reply:
Done